db/migrate/20230418075940_assing_fqdn_to_host_name.rb

```diff
@@ -0,0 +1,11 @@
+class AssingFqdnToHostName < ActiveRecord::Migration[6.1]
+  def up
+    Host.all.each do |host|
+      if !host.domain.nil? && !host.name.nil?
+        unless host.name.ends_with? host.domain.name
+          host.update(name: host.fqdn)
```

Comment:
does this trigger the orchestration? (e.g. change on the DNS server for the managed host) should we be avoiding it?

Reply:
Update does have validation so it might. I have changed that to `update_attribute` which does not trigger validation, so it should not trigger orchestration.

Reply:
ok, resolving